Accessories/uInverter/Design.md

```diff
@@ -0,0 +1,53 @@
+﻿## uInverter Design Calculations
+
+
+-   nominal dc link voltage - _12V_
+-   rated phase current - _2.5A rms_
+-   range of allowable switching frequency - _1kHz- 100kHz_
+-   range of allowable power frequency (phase frequency) - _640- 1000 Hz_
+
+ 
+### Core Components
+ #### DC Barrel Jack
+ 
+ The 2.1mm [DC barrel jack](https://www.digikey.com/en/products/detail/3643/1528-2553-ND/8346574) is rated for 15Vdc and is supplied from a standard [AC-DC wall adapter](digikey.com/en/products/detail/cui-inc/SWM30-12-NV-P5/7388285) rated for 12V. 
+
+#### DC link 
+A 3mF capacitor is selected as a DC link capacitor to protect the voltage ripple from input to the components comprising the half bridge. The capacitor is should be able to withstand a 12V input voltage. 
+
+Using formula 
+![](http://ec.kemet.com/wp-content/uploads/sites/4/2019/10/dc-link-design-tips-04-300x70.png)
+ * where _Vp(max)_ is chosen to be 0.5V
+ * duty cycle _(dc)_ assumed to be 0.75
+ * Switching frequency _fsw_ is 100kHz
+ * _Iout_ is a maximum of 7.5A 
+
+A 500mF Capacitor with a 1nF capacitor in parallel (to reduce the effect of ESR) are chosen for the DC link.
```

Comment:
Wow, a 500mF capacitor is very large!! I don't think we can put a capacitor this large on this board.

Also, does it make sense to put a 1nF cap in parallel with a 500mF cap? I agree that ESR would probably be reduced, but the relative sizes of these seems a little silly... They are 500000000x different in value. To reduce ESR (and ESL), I might recommend using a different package for the extra capacitors. Perhaps try a different shaped SMT ceramic cap, or a smaller electrolytic, or even film.

Consider using a range of caps in parallel to help with lower ESR and ESL. I don't think you need to go to such a smaller value of 1nF.